packages/shr-fhir-export/lib/extensions.js

```diff
@@ -98,7 +98,7 @@ class ExtensionExporter {
       delete(ext.publisher);
     }
     ext.contact = MVH.convertContactDetails(ext, this._config.contact);
-    ext.identifier = [{ system: this._config.projectURL, value: def.identifier.fqn }],
+    ext.identifier = [{ system: this._config.projectURL, value: def.identifier.fqn }];
```

Comment:
Nice

Reply:
It took removing and then replacing that line for me to make this monumental change.